apps/src/publicKeyCryptography/PublicKeyCryptographyWidget.jsx

```diff
@@ -107,21 +102,6 @@ const PublicKeyCryptographyWidget = React.createClass({
 });
 export default PublicKeyCryptographyWidget;
 
-/** Block of verticalbar-separated hyperlinks at top of page. */
-function HyperlinksList(props) {
-  let children = React.Children.toArray(props.children);
-  for (let i = children.length - 1; i > 0; i--) {
-    children.splice(i, 0, ' | ');
```

Comment:
No - `Array.prototype.join` creates a string, and I want an array of alternating `[Component, ' - ', Component, ' - ', Component]` here.